packages/flutter/lib/src/material/page_transitions_theme.dart

```diff
@@ -665,6 +677,7 @@ class ZoomPageTransitionsBuilder extends PageTransitionsBuilder {
   const ZoomPageTransitionsBuilder({
     this.allowSnapshotting = true,
     this.allowEnterRouteSnapshotting = true,
+    this.scrimColor,
```

Comment:
Can we set the default value here so we don't need to specifically set the color argument in [line 927](https://github.com/flutter/flutter/pull/152815/files#diff-41923961de924e865545ad350e284d38a886d56134da05aaed3e551b3640a237R927)?